<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureWallForm</class>
 <widget class="QWidget" name="PictureWallForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_main">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>10</y>
     <width>540</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_main_r">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>20</y>
     <width>540</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_main_l">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>540</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_subtract">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>80</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plus">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>80</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <zorder>label_main_r</zorder>
  <zorder>label_main_l</zorder>
  <zorder>label_main</zorder>
  <zorder>pushButton_subtract</zorder>
  <zorder>pushButton_plus</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
